credit_card.rb

```diff
@@ -1,11 +1,39 @@
-# Creadit Card
+# Credit Card
 class CreditCard
-  attr_reader :balance
-  def initialize
+  attr_reader :balance, :deposit, :withdraw
+
+  def initialize(_balance, deposit, withdraw)
```

Comment:
If you sign up for a credit card, you're usually not required to give them a balance, deposit and withdraw.  `def initialize` should not have any parameters.